README.md

```diff
@@ -190,6 +190,33 @@ $http = new Server($socket, function (ServerRequestInterface $request) {
 
 See also [example #2](examples).
 
+The `getQueryParams(): array` method can be used to get the query parameters
+similiar to the `$_GET` variable.
+
+```php
+$http = new Server($socket, function (ServerRequestInterface $request) {
+    $queryParams = $request->getQueryParams();
+
+    $body = 'The query parameter "foo" is not set. Click the following link ';
+    $body .= '<a href="/?foo=bar">to use query parameter in your request</a>';
+
+    if (isset($queryParams['foo'])) {
+        $body = 'The value of "foo" is: ' . $queryParams['foo'];
```

Comment:
Maybe we should add a note about it being a XSS vector, just to educate :stuck_out_tongue_closed_eyes: